### ISSUE TITLE
crypto.pseudoRandomBytes is deprecated

### ISSUE THREAD

**Issue by geoynomous** [expressjs/multer#811]
Using node 12.13.0 there is a DeprecationWarning:

```
DeprecationWarning: crypto.pseudoRandomBytes is deprecated.
    at DiskStorage.getFilename (/Users/test/project/node_modules/multer/storage/disk.js:8:10)
    at /Users/test/project/node_modules/multer/storage/disk.js:34:10
    at DiskStorage.getDestination (/Users/test/project/node_modules/multer/storage/disk.js:22:51)
    at DiskStorage._handleFile (/Users/test/project/node_modules/multer/storage/disk.js:31:8)
    at /Users/test/project/node_modules/multer/lib/make-middleware.js:144:17

```

**Comment by HarshithaKP** [comment]
@geoynomous  can you paste the simple recreation code for the same ?

**Comment by geoynomous** [comment]
Let me compress what I see here ...
`
const multer = require('multer');
const express= require('express');
const app = express();
app.post('/upload', multer({ dest: tmpPath, limits: { fileSize: maxFileSize, files: maxFiles }), ctrl.uploadFiles, fileErrorHandling);
`
... maybe some other middleware is involved? Will check

**Comment by gireeshpunathil** [comment]
PR to fix this:  https://github.com/expressjs/multer/pull/774

**Comment by geoynomous** [comment]
will there be a 2.0.0 release?

**Comment by LinusU** [comment]
@geoynomous this issue was fixed on the 1.x release

track 2.x status in #399

**Comment by geoynomous** [comment]
Still seeing it on the 1.4.2 release

**Comment by LinusU** [comment]
Fixed in [version 1.4.3](https://github.com/expressjs/multer/blob/master/CHANGELOG.md#143---2021-08-09) now 🚀
